components/cam/src/physics/cam/shoc.F90

```diff
@@ -107,6 +107,17 @@ module shoc
 ! Minimum TKE [m2/s2]
 real(rtype), parameter :: mintke = 0.0004_rtype
 
+!===================
```

Comment:
I am okay with declaring these as global variables since they are all parameters. A better way would be to keep these constants close to the equations which are using them unless they are needed by equations in multiple subroutines. Constants `tiny`,`fac` and `ricr` can be made global but I think rest of the constants can be defined within `pblintd_surf_temp` subroutine.